lib/passport/strategy.js

```diff
@@ -41,15 +48,17 @@ module.exports = function (context, config) {
         console.error(`Error refreshing strategy [${key}]: ${err.message}`);
       }
     });
-  };
+  }
 
-  this.verify = (accessToken, refreshToken, profile, done) => {
-    var
-      userProfile = this.config.defaultProfile || 'default',
+  verify (accessToken, refreshToken, profile, done) {
+    const
+      userProfile = this.config.defaultProfile ? this.config.defaultProfile : null,
```

Comment:
Before https://github.com/kuzzleio/kuzzle/pull/727 it would use the hardcoded value 'default'; After the PR, it will use the Kuzzle configuration `security.restrictedProfileIds`